src/App.js

```diff
@@ -1,16 +1,18 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import chatMessages from './data/messages.json';
+import ChatLog from './components/ChatLog';
 
 const App = () => {
+  const [counter, setCounter] = useState(0);
   return (
     <div id="App">
       <header>
-        <h1>Application title</h1>
+        <h1>Chat between Vladimir and Estragon</h1>
```

Comment:
What if we have senders with different names? How could we use the data from `chatMessages` to dynamically display sender names?